tests/conftest.py

```diff
@@ -64,7 +66,7 @@ def disabled_config(request, db) -> Iterator[OIDCClient]:
     yield config
 
 
-def _get_data_for_config(request: HttpRequest) -> dict[str, Any]:
+def _get_data_for_config(request: pytest.FixtureRequest) -> JSONObject:
     marker = request.node.get_closest_marker("oidcconfig")
```

Comment:
In Open Beheer, I have tried adding typing information for the markers, by doing this:

https://github.com/maykinmedia/open-beheer/blob/9704aae0f2c75b61ae499a0af6dbfdbb9a3a2d03/backend/src/openbeheer/conftest.py#L38-L47

Not sure if this would be too cumbersome to do here, but it might help to make it easier to know what can be configured?

Reply:
Added in 8994aaae0469ad1cd1f12f5ab9d0c0693e30d735